GitUI/Properties/Resources.resx

```diff
@@ -122,7 +122,7 @@
     <value>../Resources/ChangeLog.md;System.String, mscorlib, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089;Windows-1252</value>
   </data>
   <data name="Coders" xml:space="preserve">
-    <value>Henk Westhuis, Arkady Shapkin, Janusz Białobrzewski, RussKie, Steffen Forkmann, Jacob Stanley, Nick Mayer, Kevin Moore, Davide,
+    <value>Henk Westhuis, Arkady Shapkin, Janusz Białobrzewski, Igor Velikorossov, RussKie, Steffen Forkmann, Jacob Stanley, Nick Mayer, Kevin Moore, Davide,
```

Comment:
While I believe that a certain person actually deserves to be added twice, I guess it was not the intention here.
Also, Drew should take spot 5 - he is the top contributor.

Reply:
Would need to merge things like russkie's two accounts.

Reply:
```bash
git shortlog -sn | cut -c8-
```

Reply:
It appears that certain individual made commits under different names... :-\
The script gets all unique contributors and adds to the list.
I'll move Drew to the front, I missed that he's not at the helm

Reply:
Thanks for the props, but "top contributor" measured by # of commits doesn't mean much. I make lots of small commits because I find it useful after the fact when stuff (inevitably) goes wrong. The bulk of the value added to this project pre-dates my involvement.